### DISCUSSION TITLE
katex/mathjax 无法开启

### DISCUSSION THREAD

**Discussion by zirnc** [WingLim/hugo-tania#76] [Q&A] [answered]
MD 源文件：https://github.com/ChungZH/ChungZH.github.io/blob/hugo/content/articles/segment-tree.md
结果：https://blog.chungzh.cn/articles/segment-tree/

**Comment by WingLim** [comment, marked as answer]
更新主题到 https://github.com/WingLim/hugo-tania/commit/853548b9a8686aea3184cbe75bf0f39abd7f8d03 这个 commit 之后即可

之前给 `footer.html` 加了缓存，然后导致 `katex/mathjax` 参数没有正确获取，之后会把 script 单独抽出来，再加上缓存。
